addon/controllers/index.js

```diff
@@ -22,6 +23,7 @@ export default class IndexController extends Controller {
   @tracked document;
   @tracked activeGroup;
   @tracked sort;
+  @tracked listView = "true";
```

Comment:
I think you can can use a Boolean instead of a String:
```suggestion
  @tracked listView = true;
```